Cargo.toml

```diff
@@ -42,9 +42,7 @@ mint = { version = "0.5", optional = true, default-features = false }
 num-traits = { version = "0.2.14", optional = true, default-features = false }
 rand = { version = "0.7", optional = true, default-features = false }
 serde = { version = "1.0", optional = true, features = ["derive"] }
-
-[target.'cfg(target_arch = "spirv")'.dependencies]
-spirv-std = { version = "0.4.0-alpha.1", features = ["const-generics"] }
+spirv-std = { version = "0.4.0-alpha.3" }
```

Comment:
I don't really want to make this a default dependency for all glam users as it's only relevant for rust-gpu. I assume you want it on non SPIR-V targets though? Perhaps it could be enabled via a feature if that was the case?